scripts/test/AbinsLoadDMOL3Test.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import (absolute_import, division, print_function)
+import unittest
+from mantid.simpleapi import logger
+import AbinsModules
+
+
+def old_python():
```

Comment:
Now that Mantid only supports Python 2.7+ [this check](https://github.com/mantidproject/mantid/blob/81b9c3d53435f150ae0764d6475778ba24fd5a12/scripts/AbinsModules/AbinsTestHelpers.py#L42) is unnecessary and should be removed.